azure-pipelines/build-and-release.yml

```diff
@@ -5,11 +5,7 @@ stages:
 - stage: Build
   jobs:
   - job: Build
-    pool:
-        vmImage: 'windows-latest'
-        demands:
-        - Cmd
```

Comment:
Did you move these into the pool template or something?

Reply:
Ping - just want to make sure we're guaranteed to have these installed (especially npm)

Reply:
The build completes successfully. I think they are provided by default in the images. Is there something more that I need to verify?

Reply:
You should find out exactly what software is guaranteed to be installed on these images - and whether we can customize them to specify the programs we need installed. We don't want the builds failing later on because the machines aren't guaranteed to have npm installed in the future.

Reply:
I am using a compliant version of this image. It seems to come with npm pre-installed. 
https://github.com/actions/virtual-environments/blob/main/images/win/Windows2022-Readme.md

Reply:
Great, thanks!